Makefile

```diff
@@ -249,4 +248,4 @@ test-docker-push: test-docker
 	@docker push ${TEST_DOCKER_REPO}:latest
```

Comment:
@okwme do we still want this two docker commands that rely on contrib/Dockerfile.test?
The docker push, pushes 3 things, latest, branch name & commit hash. Think branch name/latest are ok.

Reply:
i'm ok with keeping as is if there's no large cost, in time or money, in doing so